src/datapane/client/config.py

```diff
@@ -117,11 +116,14 @@ def save(self, env: t.Optional[str] = None):
         if env:
             # NOTE - type checker doesn't like us setting class variables on instance
             # don't think this will cause issues
-            Config._env = env
-            Config._path = self.get_config_file(env)
+            self._env = env
+            self._path = self.get_config_file(env)
 
         with self._path.open("w") as f:
-            yaml.safe_dump(dc.asdict(self), f)
+            config_dictionary = dc.asdict(self)
```

Comment:
ok, this is def hacky lol but will do for now, I think we're hitting the limits of the dataclasses approach and want to move to using pydantic for this in the next cooldown.

Reply:
Can you just double-check and test this code-path manually - we need to be uber-cautious regarding anything hitting the config files as it hits local files on every installed users machine.

Reply:
I've run this locally and was able to login/logout/generate a report so on. The changes don't seem to break anything